combat-trainer.lic

```diff
@@ -384,6 +392,12 @@ class LootProcess
     return if Time.now - @loot_timer < @loot_delay
 
     game_state.mob_died = true
+    if @last_rites_toggle
```

Comment:
yeah - should likely.  I had set it up differently at first (using training abilities) but liked this better.  That's a lil remnant from that.